Exiled.Loader/Loader.cs

```diff
@@ -128,17 +128,13 @@ static Loader()
             .IgnoreUnmatchedProperties()
             .Build();
 
-        private static bool IsAdministrator => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
-            ? new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)
-            : geteuid() == 0;
-
         /// <summary>
         /// Runs the plugin manager, by loading all dependencies, plugins, configs and then enables all plugins.
         /// </summary>
         /// <param name="dependencies">The dependencies that could have been loaded by Exiled.Bootstrap.</param>
         public static void Run(Assembly[] dependencies = null)
         {
-            if (IsAdministrator)
+            if ((RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)) || geteuid() == 0)
```

Comment:
What happens in the case that the platform is windows and they aren't admin? The if should not be ran, but you call an unnecessary `geteuid()`.